release-notes/2.1/Preview/2.1.0-preview2-known-issues.md

```diff
@@ -26,4 +26,114 @@ We have temporarily removed the Preview 2 installers from the Linux package feed
 
 ```bash
 sudo apt install dotnet-host=2.0.6-1
-```
\ No newline at end of file
+```
+
+## Global Tools
+
+### Global Tools targeting Preview or RC will not "roll-forward" to RTM
+
+_This issue affects tool authors_
+
+Global Tools use normal .NET Core Runtime selection behavior. Runtime rules are that the specified runtime should be used if it is available. If it is not available, and the target is not a preview or RC, runtime selection rolls forward to the next available patch or, starting in .NET Core 2.1, the next minor version. However, this behavior does not apply to previews. Previews and RCs (release candidates) do not roll-forward to RTM, due to the possibility of breaking changes.
```

Comment:
Would be great to see a link here, to runtime selection behavior. If that doesn't exist at docs.ms.com, please file a docs bug.

Reply:
"RTM" is probably meaningless to the reader.